src/async-input.js

```diff
@@ -77,4 +83,5 @@ export const AsyncInputClasses = {
    * and use async-input elements.
    */
   'ASYNC_INPUT': 'i-amphtml-async-input',
+  'ASYNC_TIMEOUT': 'i-async-update-timeout'
```

Comment:
The other file thinks this is named `ASYNC_INCREASE_TIMEOUT`.

Reply:
got it